commit e0df2dce4cf6ed3850432d7e27f96bfe900e701d
Author: Reid Kleckner <rnk@google.com>
Date:   Fri Sep 6 20:12:44 2019 +0000

    Remove dead .seh_stackalloc parsing method in X86AsmParser
    
    The shared COFF asm parser code handles this directive, since it is
    shared with AArch64. Spotted by Alexandre Ganea in review.
    
    llvm-svn: 371251

diff --git a/llvm/lib/Target/X86/AsmParser/X86AsmParser.cpp b/llvm/lib/Target/X86/AsmParser/X86AsmParser.cpp
index a061b0485bf..54877adf800 100644
--- a/llvm/lib/Target/X86/AsmParser/X86AsmParser.cpp
+++ b/llvm/lib/Target/X86/AsmParser/X86AsmParser.cpp
@@ -874,7 +874,6 @@ private:
   bool parseSEHRegisterNumber(unsigned RegClassID, unsigned &RegNo);
   bool parseDirectiveSEHPushReg(SMLoc);
   bool parseDirectiveSEHSetFrame(SMLoc);
-  bool parseDirectiveSEHAllocStack(SMLoc);
   bool parseDirectiveSEHSaveReg(SMLoc);
   bool parseDirectiveSEHSaveXMM(SMLoc);
   bool parseDirectiveSEHPushFrame(SMLoc);
@@ -3819,19 +3818,6 @@ bool X86AsmParser::parseDirectiveSEHSetFrame(SMLoc Loc) {
   return false;
 }
 
-bool X86AsmParser::parseDirectiveSEHAllocStack(SMLoc Loc) {
-  int64_t Size;
-  if (getParser().parseAbsoluteExpression(Size))
-    return true;
-
-  if (getLexer().isNot(AsmToken::EndOfStatement))
-    return TokError("unexpected token in directive");
-
-  getParser().Lex();
-  getStreamer().EmitWinCFIAllocStack(Size, Loc);
-  return false;
-}
-
 bool X86AsmParser::parseDirectiveSEHSaveReg(SMLoc Loc) {
   unsigned Reg = 0;
   int64_t Off;
